### PR TITLE
Remove duplicates from reporting

### PR DESCRIPTION
**Pull request by johannah-pik**
Some variables are reported based on the EDGE-T results AND based on the REMIND results. As there is currently no differentiation in the variable naming - the affected variables are moved to the extended reporting. As a result, there are no longer duplicates in the standard reporting after an iterative run but the reporting after a stand alone run delivers the full range of variables.

### CODE REVIEW

File: R/reportEDGET.R [hunk 1]
```diff
@@ -963,6 +934,44 @@ toolReportEDGET <- function(output_folder = ".",
     toMIF[period == 2100][, period := 2150]
   )
 
+  varsl <- list(
```
Comment (dklein-pik) [inline R/reportEDGET.R:937]: Hi Johanna, sorry to meddle here. I am probably missing something, but to me it seems that the variables were not moved into the extended reporting but below it and would still be part of the not extended reporting. Is that right?

**Comment by dklein-pik** [comment]
> Some variables are reported based on the EDGE-T results AND based on the REMIND results. As there is currently no differentiation in the variable naming - the affected variables are moved to the extended reporting. As a result, there are no longer duplicates in the standard reporting after an iterative run but the reporting after a stand alone run delivers the full range of variables.

Does this PR close the issue remindmodel/remind/issues/1073 finally, or are you planning to rename the variables later?

File: R/reportEDGET.R (hunk 1 above)
Comment (johannah-pik) [reply R/reportEDGET.R]: Hi David- thanks for finding that stupid bug. I will test it again on a REMIND run to make sure that all duplicates are gone with the fix

**Comment by johannah-pik** [comment]
> > Some variables are reported based on the EDGE-T results AND based on the REMIND results. As there is currently no differentiation in the variable naming - the affected variables are moved to the extended reporting. As a result, there are no longer duplicates in the standard reporting after an iterative run but the reporting after a stand alone run delivers the full range of variables.
> 
> Does this PR close the issue [remindmodel/remind/issues/1073](https://github.com/remindmodel/remind/issues/1073) finally, or are you planning to rename the variables later?

I am planning to rename the variables- so you can leave it open for now.

**Review by dklein-pik — APPROVED** [review]

**Review by MariannaR — APPROVED** [review]

**Pull request merged**